crates/types/src/ycbcr422_image.rs

```diff
@@ -25,12 +25,34 @@ pub struct YCbCr422Image {
     buffer: Arc<Vec<YCbCr422>>,
 }
 
+impl From<RgbImage> for YCbCr422Image {
+    fn from(rgb_image: RgbImage) -> Self {
+        Self {
+            width_422: rgb_image.width() / 2,
+            height: rgb_image.height(),
+            buffer: Arc::new(buffer_422_from_rgb_image(rgb_image)),
+        }
+    }
+}
+
+impl From<&YCbCr422Image> for RgbImage {
+    fn from(val: &YCbCr422Image) -> Self {
```

Comment:
Value? Vallejo (Amtrak station code; Vallejo, CA)? Valley? Valve? Valerie? Valid? Validation? Validate? Valine (Amino Acid)? Valedictorian? Véhicule Automatique Léger (French driverless metro system)? Viking Air Limited (Canada)? Vatican City Lira (defunct currency; ISO designation)? Value Added Logistics (supply chain)? Voice Activated Lightstand (photography)? Variable Angle Launcher? Vertical Alarm Limit? Value-Oriented Algorithmic Language?

Please don't abbreviate. Also in the method below.

What is it? An image, right? Why not name it like that? :wink:

Reply:
Quite true :D but the list is epic xD I'll fix it.

Reply:
Fixed.